### ISSUE TITLE
Fixing Process #1

### ISSUE THREAD

**Issue by BerriJ** [BerriJ/UnitCircle#3]
!! Carefully consider the editorial rules: http://www.iaria.org/editorialrules.html
!! Have a carefully proofreading the updated version; an English specialist, or a native English speaker is recommended.
!! Carefully observe the required format (see http://www.iaria.org/format.html) on text style and size, captions, references, etc.
!! More detailed editorial comments will come after the first update
Note
- editorial (format, punctuation) and English comments are mandatory, as imposed by the Publisher
- content comments pointing to mistakes/errors should be fixed or statement removed to keep the quality of the paper high
- content comments which are not the real target or asking more difficulty to be added should be kindly indicated in CONCLUSION AND FUTURE WORK with some technical details; this will show a 'thank you to those suggesting them', but also showing your strengths.

# reviewer 1

The paper is really nice. The area you need to amend is the beginning of the abstract. 

- [x] Do you use "will" but rather the paper does so and so, it is done already and no longer a future tense.
- [ ] It is also very necessary that you define all equations used in the article. All the equations must also be made referenced to in the content of the article to show their reference in the session used.
Paper topic is quite interesting. The use of ANN-Data Analytics is a well-known topic, so the main paper contribution seems to be it application to WiFi quality evaluation. 
- [ ] In this sense, the obtained conclusions are pretty simple and very vague and must be tuned.

Authors discuss the use of self-organized learning neural networks to analyze the user experience about the WiFi network provided on German high speed trains in different areas of Germany.

- [ ] Authors should provide a detailed discussion about the relationships with related works and provide a discussion about possible alternatives to the use of self-organized learning neural networks and compare the results of the different solutions.

=> Position Data analysis with classical statistical approaches is quite complex.

#  reviewer 2

This paper presents an analysis of the wifi connections of the German high-speed trains - InterCityExpress (ICE). The analysis is announced to focus on the user experience ("To be more specific we analyse the user experience when using the WiFi network that is provided on German high-speed trains (ICEs) in different regions of Germany"). 

- [x] However, results only show wifi parameters analysis. The paper is not clear about this: do the trace include the user feedbacks and the authors try to correlate them with wifi parameters ? Or do the authors assume that the user experience is totally depending on wifi parameters?

--> We assumed that the user experience using the WiFi mainly depends on the latency and the download rate. To proof this one could analyse the correlation between data on the user experience and our results. Unfortunately we where not able to gather data on the user experience so this remains an interesting area for subsequent research.

- [ ] Authors use and emphasize a Self-Enforcing Network and R-studio tools to analyze the dataset; are there *other classification methods* that can obtain the same results ? What are specifics metrics that only these tools can achieve ?

"While the identification of areas where the supply of mobile internet access is good or bad was possible, we were not able to identify an underlying pattern which explains this completely." How could it be otherwise? Mixing macro-mobility (for the high speed trains) and micro-deployment of antennas can not result in macro results depending only on the geographical train mobility. 
- [x] The trace is missing from external antennas parameters (distance between antennas, signal strength, interferences, load, etc). Complete them in camera-ready version.

# reviewer 3

The paper aims to present an analysis of the WiFi quality on German highspeed trains.

- [ ] This paper analyses data; please bring some meaningful conclusion/pattern. Stress-out the scientific contribution. 
- [ ] This study is limited to the sole case of the German trains. can this be generalized.
- [x] On the other hand, English could be improved. Indeed, the paper contains some typos (e.g., in the title, the word identify should start with an upper-case; high relevance for -> high relevance of, etc.). 
- [x] The presentation could be improved as well. In particular, the sub-sections of the Section III and of the Section IV could be numbered properly.

# reviewer 4

- [ ] It would be great if the paper explains more about the reasons for the particular activation function in (2); process to adjust and identify the presented CVF values in Table II. For example, In "until a clear grouping of the objects is reached", how to determine whether a clear grouping has been done or not?

- [ ] Under Preprocessing the Data section, what are the 2 variables? ("In 2 variables the amount of missing observations...")

# reviewer 5

The authors made a study about the quality of WiFi data from the German highspeed trains. The goal of the authors with this study is to indicate where the network coverage is especially good or bad and suggest ways to improve its quality. The study is very relevant as is explained in the article.
The article is well written, presented and has a good related work section. 

However, some questions have arisen throughout its reading.

- [ ] The authors use a self-organized learning neural network to analyze data, however it is not clear, or well justified why this kind of network. Do you have tried other types of networks, recurrent networks, ...? 
- [ ] Also, the choice of the logarithmic-linear activation function plays an important role, but it is not well justified.
- [ ] How many clusters do you reach in Figure 4? How are observations, regions selected from this clustering? It is not clear the utility of the data segmentation, if your study and results are made in two particular well defined lines: ICE Line 10 and 26.

# reviewer 6

- [ ] While the paper is clear with the motivation and objectives of the work, it is not as detailed with the methodology used. Why not use more space (they have 1 remaining page) explaining why these weights in table II for these attributes were reached? 
- [ ] Why does the location (latitude and longitude) receive different weights between the two clustering processes? Some discussion is needed around this.

# reviewer 7

This is an interesting and well-written paper in the area of the conference. I recommend acceptance of this paper subject to minor modifications.

- [x] p.1.c.1.l.10 ... are discussed in numerous recent publications, using very different ... [5],[9],[10],[11],[12],[13] ...

- [x] c.2.l.12 ... WiFi on the trains, ... the data used ...
l.-8 ... the weight matrix displays ... [19],[20],...

- [x] p.2.c.1.l.4 follows
- [x] l.9 Indent this paragraph
- [x] l.-17 ... and is sent to different ...
- [x] l.-3 Suggest to put variables in italics. So put 'sid' in italics.

- [x] c.2 Table I. Put variables in italics.
- [x] l.-10 'sid' in italics

- [x] p.3.c.1.l.10 ... because it is evidence that the data is not uniformly distributed ...

- [x] c.2 Table II. Put variables in italics.

- [x] p.4.c.2.l.8. Indent this paragraph
- [x] l.16 ... region is not highly populated ...
- [x] l.-19 ... they contrast with most observations on other big cities ...

# reviewer 8

The abstract is well written and provides a clear overview of the contribution of the paper.
The first paragraph of the introduction is very general. You could perhaps focus on the specific paper more quickly. The abstract focuses on wifi quality but the opening of the introduction focuses on factors affecting demand for wifi. 
- [ ] Paragraphs 2 and 3 of the introduction could perhaps be switched around, introducing the case first and then the existing literature in the area that is relevant to the case.
The methodology is well described and the city cases are interesting; however, the analysis of the data is limited. 
- [ ] The conclusions lack content. In the abstract you say that wifi quality can be explained by ‘just a few determinants’ and that you will show ways of improving the wifi quality. This is not evident in the conclusion. The conclusion states where they is good and bad wifi but you have not said what the determinants are or how to improve the quality. Perhaps reduce the description of the cases slightly so there is more opportunity to extract and synthesize the key points to deliver the contribution promised in the abstract.

# reviewer 9

The paper formatting needs some adjustments:

- [x] !! in the affiliation section, add the label ‘email:’ in front of the address

- [x] !! Reduce the spacing between text paragraphs, all across the paper

- [x] !! Reduce the spacing above each section title
*Nicht umgesetzt da in der Vorlage exakt so wie bei uns*

- [x] !! avoid ALL the footnotes:
- [x] - if URLs, put them at References, refer to them via [x], and provide the date of the last access for ALL online references
*The provieded Citation Style doesn't have a field for "last accessed" or similar.*

- [x] - if text only, embed them into the paper (see pages 1, 2)
* Not clear what that means*

- [x] !! Number all subsection titles -> use capital letters; for example, correct it:
-- Data
Must be:
--> A. Data
!! Correct all

- [x] !! Correct the Table caption:
-- TABLE I. Selected variables.
--> TABLE I. SELECTED VARIABLES.
!! Place the table captions above, not below the tables
!! Write all words in UPPERCASE in each table caption
!! Apply to all
*Captions aren't uppercase in the template so we have just moved them above the tables.*

- [x] !! Reduce the spacing below each figure and table
*We are just using the template. We removed some extra space that was added by us*

## correct the format for all the References (see template):

- [x] do not underline the web addresses
- [x] provide the last access date for ALL on-line references in the form [retrieved: month, year] (close to the camera-ready deadline) -> correct: [1], [2], [4], etc.

- [x] list all the authors, if less than 6 [otherwise, use 'et al.' after the first author name] -> correct: [7], [10].

- [x] write all titles in English -> correct: [18]
*It's a presentation from the DeutscheBahn that isn't available in english.*

- [x] Correct the format / grammar / punctuation errors. Some examples:
-- Abstract—In this paper we will demonstrate
Must be:
--> Abstract—In this paper, we will demonstrate
!! always this way, in similar constructions, across the whole paper

- [x] -- In recent years the demand for
--> In recent years, the demand for

- [x] -- for mobile internet access
--> for mobile Internet access

- [x] -- in various ways (e.g. [1], [2]).
--> in various ways (e.g., [1][2]).
!! hereafter, across the entire paper; always e.g., and i.e., have a ',' before (when inside a sentence) and after

- [x] -- rise in demand [3], [4].
--> rise in demand [3][4].

- [x] -- networks in general
[5], [6]
--> networks in general [5][6]
!! Correct all citations across the paper

- [x] -- In an increasingly competitive transportation industry the existence of
--> In an increasingly competitive transportation industry, the existence of

- [x] -- In
section III the technology
--> In Section III, the technology
!! Capitalize word 'Section' in all such comments across the paper

- [x] -- SEN are found in e.g. [19], [20].
--> SEN are found in, e.g., [19][20].

- [x] -- In this case the semantical
--> In this case, the semantical

- [ ] !! expand all the acronyms, even trivial, when firstly used (even in the abstract); examples: SSID, etc.
*SSID ist angepasst. Finde keine weiteren Abkürzungen*

- [x] !! Do not insert web addresses in text; instead, insert them as references; example:
-- open-data-portal (https://data.deutschebahn.com/).
--> open-data-portal [xx].

- [x] -- Furthermore the rate of downloads
--> Furthermore, the rate of downloads

- [x] -- downloaded faster e.g. a website
--> downloaded faster, e.g., a website

- [x] -- Therefore we looked
--> Therefore, we looked

- [x] -- On the contrary there is high
--> On the contrary, there is high

- [x] -- In the preceding chapters we showed
--> In the preceding sections, we showed

- [x] -- To visualize our results we used
--> To visualize our results, we used

- [ ] etc., etc.
